pdr_backend/util/csvs.py

```diff
@@ -0,0 +1,36 @@
+import os
+import csv
+
+
+def write_csv(all_predictions, csv_output_dir):
```

Comment:
Rename to `write_prediction_csv()`. We will surely have other csvs.

Also, as mentioned in the github issue, it points to a pattern in df-py csvs how to have function names for writing, reading, and naming the csvs; and accompanying unit tests. We should have that here.

Reply:
done